src/style.css

```diff
@@ -1,3 +1,100 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Don't use * selector for zeroing out your margins or paddings. It's still inefficient for browser to read your web document